src/ui/SortDropdown/SortDropdown.ts

```diff
@@ -67,14 +67,16 @@ export class SortDropdown extends Component {
     this.buildDropdown();
   }
 
-  private clearDropdown() {
+  private clearDropdown(): boolean {
+    const hasFocus = this.dropdown && document.activeElement === this.dropdown.getElement();
     this.dropdown && this.element.removeChild(this.dropdown.getElement());
     this.dropdown = null;
+    return hasFocus;
   }
 
   private buildDropdown() {
     this.sortComponents = this.getEnabledSortComponents();
-    this.clearDropdown();
+    const hasFocus = this.clearDropdown();
```

Comment:
Not a big deal, but it seems slightly odd to me that `clearDropdown` returns a boolean.

We could just do:

```typescript
const hasFocus = this.dropdown && document.activeElement === this.dropdown.getElement();
this.clearDropdown();
// ...
```